clang/lib/Driver/Driver.cpp

```diff
@@ -816,8 +816,10 @@ static bool isValidSYCLTriple(llvm::Triple T) {
       T.getOS() == llvm::Triple::CUDA && !T.hasEnvironment())
     return true;
 
-  // AMDGCN is valid for SYCL
-  if (T.isAMDGCN())
+  // 'amdgcn-amd-amdhsa' is the valid SYCL triple for AMD GPUs.
```

Comment:
Can https://github.com/intel/llvm/blob/0e3e6d234a94d7d54bba21ae7a7e43f500812b1d/sycl/doc/UsersManual.md?plain=1#L42 be updated to reflect the expected triple to be used here?

Reply:
Updated :+1: